lldb/source/Breakpoint/Breakpoint.cpp

```diff
@@ -838,6 +840,14 @@ void Breakpoint::GetDescription(Stream *s, lldb::DescriptionLevel level,
                                 bool show_locations) {
   assert(s != nullptr);
 
+  // Grey out any disabled breakpoints in the list of breakpoints.
+  const bool highlight_disabled =
```

Comment:
Suggestion: rename this variable to "dim_breakpoint_description" or something like that.
The word "highlight_disabled" could also be applied to situations where colors are not enabled.